app/components/rdf-form-fields/rich-text-editor.hbs

```diff
@@ -28,28 +28,32 @@
   <div class="rich-text-editor" {{on "focusout" this.updateValue}}>
     <EditorContainer>
       <:top>
-        <Toolbar as |Tb|>
-          <Tb.Group>
-            <Plugins::Heading::HeadingMenu
-              @controller={{this.editorController}}
-            />
-          </Tb.Group>
-          <Tb.Group>
-            <Plugins::TextStyle::Bold @controller={{this.editorController}} />
-            <Plugins::TextStyle::Italic @controller={{this.editorController}} />
-            <RdfFormFields::RichTextEditor::ExtraFormattingOptions
-              @controller={{this.editorController}}
-            />
-          </Tb.Group>
-          <Tb.Group>
-            <Plugins::List::Unordered @controller={{this.editorController}} />
-            <Plugins::List::Ordered @controller={{this.editorController}} />
-          </Tb.Group>
-          <Tb.Group>
-            <Plugins::Link::LinkMenu @controller={{this.editorController}} />
-            <Plugins::Table::TableMenu @controller={{this.editorController}} />
-          </Tb.Group>
-        </Toolbar>
+        <ResponsiveToolbar>
+          <:main as |Tb|>
+            <Tb.Group>
+              <Plugins::Heading::HeadingMenu
+                @controller={{this.editorController}}
+              />
+            </Tb.Group>
+            <Tb.Group>
+              <Plugins::TextStyle::Bold @controller={{this.editorController}} />
+              <Plugins::TextStyle::Italic @controller={{this.editorController}} />
+              <RdfFormFields::RichTextEditor::ExtraFormattingOptions
+                @controller={{this.editorController}}
+              />
+            </Tb.Group>
+            <Tb.Group>
+              <Plugins::List::Unordered @controller={{this.editorController}} />
+              <Plugins::List::Ordered @controller={{this.editorController}} />
+            </Tb.Group>
+            <Tb.Group>
+              <Plugins::Link::LinkMenu @controller={{this.editorController}} />
+            </Tb.Group>
+            <Tb.Group>
+              <Plugins::Table::TableMenu @controller={{this.editorController}} />
```

Comment:
Something is still wrong with the styles here, we probably need to import extra styles.

Reply:
This is also the only real user-facing change. The component added extra options to change the background color and alignment of the table cells. Not sure if we actually want that but the component doesn't have an option to hide them either.

Reply:
I think this is fine to keep